src/Psy/Configuration.php

```diff
@@ -772,6 +774,26 @@ public function getLoop()
     }
 
     /**
+     * Enables tab completion mode
+     *
+     * @param $tabCompletion
+     */
+    public function setTabCompletion($tabCompletion)
+    {
+        $this->tabCompletion = (bool) $tabCompletion;
```

Comment:
(sorry, meant to leave that comment on `getTabCompletion`, not `setTabCompletion`)